docs/user/set-up/install-agent-binary.md

```diff
@@ -0,0 +1,32 @@
+---
+aliases:
+- /docs/agent/latest/set-up/install-agent-binary/
+title: Install the Grafana Agent binary
+weight: 140
+---
+
+## Install the Grafana Agent binary
+
+Install Grafana Agent and get it up and running using a binary file. 
+
+### Overview
```

Comment:
I've not been using 'overview' for intro material. Should I be? Is this a change to our writing guidelines?

Reply:
same comment as below